tests/tests_nn/test_rim.py

```diff
@@ -144,11 +143,15 @@ def test_rim(
 )
 @pytest.mark.parametrize(
     "image_init",
-    [
-        "zero-filled",
-        "sense",
-        "input-kspace",
-    ],
+    ["zero_filled", "sense", "input_kspace", "input_image"],  #  None
+)
+@pytest.mark.parametrize(
+    "learned_initializer",
+    [True, False],
+)
+@pytest.mark.parametrize(
+    "input_image_is_None",
+    [False], # True - not yet supported
```

Comment:
Let's test OpenVINO inside the existing test. It's not very elegant in terms of test development but simples for support.